proposals/NNNN-SDL-behavior-in-case of-Low-Voltage.md

```diff
@@ -0,0 +1,74 @@
+# SDL behavior in case of LOW_VOLTAGE event
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / Web / RPC / Protocol]
+
+## Introduction
+
+This proposal defines SDL behavior in case of LOW_VOLTAGE event.
+LOW_VOLTAGE scenario is triggered when the battery voltage hits below 7v. In case of such event EMMC is turned off and all read/write operations are unavailable. After the voltage level is restored all oprations are resumed.
+
+## Motivation
+
+Implement logic that will allow SDL to resume after battery charge is restored or to start up correctly in the next ingnition cycle if SDL was shut down due to LOW VOLTAGE event. 
+
+## Proposed solution
+
+When battery voltage hits below 7v SDL will "freeze" all operation untill it will be switched off or resumed. During LOW_VOLTAGE state proposed the following SDL behavior:
+
+- SDL ignores all requests from mobile applications
+- SDL ignores all responses and messages from HMI except messages for "WAKE_UP" or "IGNITION_OFF"
+- SDL stops audio/video streaming
+- During LOW_VOLTAGE all transports are unavailable for SDL
+- SDL persists resumption related data stored before receiving LOW_VOLTAGE message
+- After WAKE_UP applications data will be resumed to the state before LOW_VOLTAGE event
```

Comment:
@dboltovskyi please remove ` resumed to the state before LOW_VOLTAGE event`
Because state is not the same. All application will be unregistered and device disconnected

Reply:
Done